gtk-subclass/tests/simple_application.rs

```diff
@@ -209,5 +219,15 @@ fn test_create() {
 
     application.connect_activate(|_| {});
 
+    gtk::timeout_add_seconds(1, clone!(application => move || {
```

Comment:
Can we not have a 1s timeout here but somehow trigger this directly when the window is shown? There should be a signal on the window